Source/Core/DiscIO/NANDContentLoader.h

```diff
@@ -17,6 +17,34 @@ namespace DiscIO
 {
 bool AddTicket(u64 title_id, const std::vector<u8>& ticket);
 
+class CNANDContentData
+{
+public:
+	virtual const std::vector<u8> Get() = 0;
+	virtual bool GetRange(u32 start, u32 size, u8* buffer) = 0;
+};
+
+class CNANDContentData_File final : public CNANDContentData
```

Comment:
It's more supposed to be a private sort of thing (not exposed directly, only thru the base class), but I couldn't find a decent place to hide them at. I can remove the underscore tho if you prefer that.